.eslintrc.js

```diff
@@ -40,8 +47,8 @@ module.exports = {
     // Style
     quotes: ['error', 'single', { avoidEscape: true }],
 
-    // ensures clean diffs, see https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8
-    'comma-dangle': ['error', 'always-multiline'],
+    // Covered by prettier
+    'comma-dangle': 'off',
```

Comment:
Specifically adding in the `'comma-dangle': off` clashes with prettier when linting and does not enforce that dangling commas should be there. 

When using eslint with prettier, I think it would be better to add in the prettier plugin to our `.eslintrc.js` as specified [here](https://dev.to/s2engineers/how-to-make-eslint-work-with-prettier-avoiding-conflicts-and-problems-57pi), and adding it at the end of our array of plugins.

And for things that are already covered by prettier like comma-dangle, instead of specifying off, we can just not mention the `comma-dangle` property at all, as comma-dangle off doesn't enforce dangling commas when linting within our pre-commit hook or `yarn lint-fix` command, but commenting it out does enforce it.

Reply:
kk sounds good, ty for the review!